tests/cases/fourslash/stringLiteralCompletionsInJsxAttributeInitializer.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path="fourslash.ts" />
+
+// @jsx: preserve
+// @filename: /a.tsx
+////type Props = { a: number } | { b: "b" };
```

Comment:
can you change this to something else so it's obvious that the suggestion is the initialiser, not the property name?
```suggestion
////type Props = { a: number } | { b: "somethingelse" };
```